transports/tcp/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 0.39.0 [unreleased]
 
 - Update to `libp2p-core` `v0.39.0`.
+- Fix `Transport::remove_listener` [PR 3387](https://github.com/libp2p/rust-libp2p/pull/3387).
```

Comment:
Please use a link reference like in the other entries.

Also, can you include here what is actually being fixed?

Like:

```suggestion
- Fix a bug where we removed any other listener in `Transport::remove_listener` except for the one with the provided `ListenerId`. See [PR 3387].
```